src/common.cc

```diff
@@ -28,7 +28,11 @@ namespace sharp {
   bool IsTiff(std::string const &str) {
     return EndsWith(str, ".tif") || EndsWith(str, ".tiff") || EndsWith(str, ".TIF") || EndsWith(str, ".TIFF");
   }
-
+#ifdef HAS_OPENSLIDE
```

Comment:
We probably only need to use the `#ifdef HAS_OPENSLIDE` wrapper around code that won't otherwise compile, e.g. when calling `vips_openslideload`.